src/listeners/addListeners.ts

```diff
@@ -0,0 +1,55 @@
+import { constants } from '../data/constants';
+import { applyPromoCode } from '../pages/basket/listeners/applyPromoCode';
+import { clearTheCart } from '../pages/basket/listeners/cartCleaner/clearTheCart';
+import { doNotClearTheCart } from '../pages/basket/listeners/cartCleaner/doNotClearTheCart';
+import { reallyClearTheCart } from '../pages/basket/listeners/cartCleaner/reallyClearTheCart';
+import { plusMinusOneProduct } from '../pages/basket/listeners/changeQuantity/plusMinusOneProduct';
+import { copyPromoCode } from '../pages/basket/listeners/copyPromoCode';
+import { removeProductFromCart } from '../pages/basket/listeners/removeProductFromCart';
+import { addCategoryLink } from '../pages/catalog/listeners/addCategoryLink';
+import { addProductLink } from '../pages/catalog/listeners/addProductLink';
+import { addProductToCartFromCatalog } from '../pages/catalog/listeners/addProductToCartFromCatalog';
+import { addSubLink } from '../pages/catalog/listeners/addSubLink';
+import { filterProductList } from '../pages/catalog/listeners/filterProductList';
+import { getModal } from '../pages/catalog/listeners/getModal';
+import { paginateModal } from '../pages/catalog/listeners/paginateModal';
+import { removeModal } from '../pages/catalog/listeners/removeModal';
+import { searchByButton, searchByEnter } from '../pages/catalog/listeners/searchProduct';
+import { showFilter } from '../pages/catalog/listeners/showFilter';
+import { sortByValue } from '../pages/catalog/listeners/sortProducts';
+import { switchPageByHashChain } from '../pages/catalog/listeners/switchPageByHashChain';
+import { switchPhoto } from '../pages/catalog/listeners/switchPhoto';
+import { viewNextPage } from '../pages/catalog/listeners/viewNextPage';
+import { viewPrevPage } from '../pages/catalog/listeners/viewPrevPage';
+import { submitLogin } from '../pages/logReg/validation/authorizationFunctions/submitData/submitLogin';
+import { submitReg } from '../pages/logReg/validation/authorizationFunctions/submitData/submitReg';
+
+export function addListeners() {
+    submitLogin(constants.authorization);
```

Comment:
create 1 listener for click, because you just check the class name, if one class call one function, other class - ther function, but listener will be the one, because when click occurs it triggers all the same listeners, you need to call I listener which will delegate work to particular function